src/main/java/org/truffleruby/Main.java

```diff
@@ -63,11 +63,13 @@
 
     public static final String LANGUAGE_ID = "ruby";
     public static final String LANGUAGE_VERSION = "2.3.3";
-    public static final String PREFIX = "truffleruby.";
     public static final String BOOT_SOURCE_NAME = "main_boot_source";
     public static final String RUBY_COPYRIGHT = "truffleruby - Copyright (c) 2013-2017 Oracle and/or its affiliates";
-    private static final boolean METRICS_TIME = Boolean.getBoolean(PREFIX + "metrics.time");
-    private static final boolean METRICS_MEMORY_USED_ON_EXIT = Boolean.getBoolean(PREFIX + "metrics.memory_used_on_exit");
+
+    // These system properties are used before outside the SDK option system
+
+    private static final boolean METRICS_TIME = Boolean.getBoolean("truffleruby.metrics.time");
+    private static final boolean METRICS_MEMORY_USED_ON_EXIT = Boolean.getBoolean("truffleruby.metrics.memory_used_on_exit");
```

Comment:
I assume we cannot be consistent and use the `polyglot.ruby.` prefix here because otherwise the Engine would complain about unknown options?